example.js

```diff
@@ -0,0 +1,28 @@
+var Hapi = require('hapi')
+var PouchDB = require('pouchdb-core')
+  .plugin('pouchdb-plugin-http')
```

Comment:
`.plugin(require('pouchdb-plugin-http'))`

also add `pouchdb-replication` and `pouchdb-mapreduce`